prep-javascript-loops/main.js

```diff
@@ -0,0 +1,69 @@
+function whileLoop1() {
+  var Array = [];
```

Comment:
all of the Array variables should be changes to `array` and you should declare them with the const keyword.